editor/styling/stylesheets/components/_tab.scss

```diff
@@ -35,9 +32,6 @@
 
             .tab-label {
                 -fx-text-fill: -df-text-selected;
-                .image-view {
-                    @include effect-lighten();
-                }
```

Comment:
This rule and a similar one above had no effect on the editor. We have 2 tab panes with images in the editor:
- resource views, which are already styled in a different place with special colors for different resources
- tools tab pane with Console, Curve Editor, etc, but those images are not image views, instead, those are implemented as background images